textxjinja/__init__.py

```diff
@@ -62,10 +63,21 @@ def eval_file_name(file_name):
             elif ph_value is True:
                 file_name = file_name.replace(placeholder, '')
             elif ph_value is not None:
-                file_name = file_name.replace(placeholder, ph_value)
+                if isinstance(ph_value, list):
```

Comment:
`ph_value` is used in the for loop iteration few lines bellow. So this test is a bit strict. It could be relaxed by checking if `ph_value` is iterable. There are two popular techniques for that: using `iter` and catching `TypeError` exception or using `isinstance` with `collection.abs.Iterable` (see [this SO post](https://stackoverflow.com/questions/1952464/in-python-how-do-i-determine-if-an-object-is-iterable)).

Reply:
Original implementation was with `iter` but it doesn't work in all cases since string is iterable. So I've added an extra check to skip if variable type is string.